draft-ietf-privacypass-auth-scheme.md

```diff
@@ -140,23 +140,30 @@ in more detail.
 ## Token Challenge {#challenge}
 
 Origins send a token challenge to clients in an "WWW-Authenticate" header field
-with the "PrivateToken" scheme. This challenge includes a TokenChallenge
-message, along with information about what keys to use when requesting a token
-from the issuer.
+with the "PrivateToken" scheme. This field is a series parameters, each a key-value pair. There are two mandatory parameters:
+one containing a token challenge and another containing the token-key used for
+producing (and verifying) a corresponding token. The challenge parameter can be
+repeated in the header field.
```

Comment:
The challenge parameter isn't repeated, the entire authentication challenge with all parameters is repeated. Again, this is just how auth schemes work.

```suggestion
producing (and verifying) a corresponding token.
```

Reply:
Right, right.